pkg/sfu/session.go

```diff
@@ -204,6 +204,15 @@ func (s *Session) OnClose(f func()) {
 	s.onCloseHandler = f
 }
 
+func (s *Session) SendMessage(method, params, origin string) {
```

Comment:
Might be better named BroadcastAPIChannelMessage for the whole Session

Reply:
Agree. I wasn't too happy with my naming :-)